validator/main.go

```diff
@@ -693,6 +693,9 @@ func isDriverManagedByOperator(ctx context.Context) (bool, error) {
 
 func validateHostDriver(silent bool) error {
 	log.Info("Attempting to validate a pre-installed driver on the host")
+	if fileInfo, err := os.Lstat("/host/usr/bin/nvidia-smi"); err == nil && fileInfo.Size() == 0 {
```

Comment:
So if the file is not found, we should throw an error. This logic would be in an `else if` clause

Reply:
Updated.